db/re/item_db.txt

```diff
@@ -4953,7 +4956,8 @@
 6960,Key_Air_Fortress,Key Air Fortress,3,10,,10,,,,,,,,,,,,,{},{},{}
 6961,Huge_Metal_Scrap,Huge Metal Scrap,3,10,0,1000,,,,,,,,,,,,,{},{},{}
 6962,Old_Fuel,Old Fuel,3,10,0,100,,,,,,,,,,,,,{},{},{}
-
+6963,HPdrainStone_Robe,HP Absorption Stone(Garment),6,10,,,,,,,,,,,,,,,{ bonus2 bHPDrainRate,10,1; },{},{}
```

Comment:
Those stone should have weight = 10
according to KRO page

https://www.divine-pride.net/database/item/6963
https://www.divine-pride.net/database/item/6964